assets/dev/js/editor/utils/controls-css-parser.js

```diff
@@ -37,6 +37,33 @@ ControlsCSSParser = elementorModules.ViewModule.extend( {
 		} );
 	},
 
+	updateBackgroundImagesSizes( control, values ) {
+		if ( control.name.includes( 'background_image' ) ) {
```

Comment:
This is the file corresponding to core/files/css/base.php of the editor fronted. 
Also talk about it with Mati. This is the location of the logic should be.

Reply:
The Controls CSS Parser is almost entirely generic (except for some Globals handling).
I would not put specific control handling here.

Reply:
To add to that - you cannot use a hard-coded prefix here, since each Background Group Control has a different prefix.
Take the Flip Box widget for example, it has multiple Background Group Controls with different prefixes (`background_a`, `background_b`).

You will need to use a different control model property to identify it as a background control.

You might need to add a property (e.g. `'is_background_image'`) to the `'image'` field in `background.php` which you could then use in the CSS parser to identify it.

Reply:
Not relevant anymore, the logic has changed.